spark/dl/src/main/scala/com/intel/analytics/bigdl/tensor/TensorNumeric.scala

```diff
@@ -31,6 +31,8 @@ class TensorNumericMath
  */
 object TensorNumericMath {
 
+  type NumericWildcard = Any
```

Comment:
Please add a detailed document to explain why we set this and when should it be use.